tests/tests.sh

```diff
@@ -33,7 +33,7 @@
 #
 ############################################################################################
 
-all_tests=$(find tests -name "*.dk")
+all_tests=$(find tests -name "*.dk" | sort)
```

Comment:
It is a ugly hack to rely on the alphabetical order to encode the dependency order between files.

Reply:
Yeah, but this way it is deterministic and it works.